contrib/brl/bbas/bpgl/algo/bpgl_bundle_rolling_shutter_adjust.cxx

```diff
@@ -136,7 +136,6 @@ bpgl_bundle_rolling_shutter_adj_lsqr::f(vnl_vector<double> const& a,
                                         vnl_vector<double> const& b,
                                         vnl_vector<double>& e)
 {
-  typedef vnl_crs_index::sparse_vector::iterator sv_itr;
```

Comment:
Commit name misspelling `usued`->`unused`.

Reply:
done in all 4 files